### PR TITLE
Use New Applications Endpoints

### PR DESCRIPTION
**Pull request by dimak1**
*Issue:*
- https://github.com/bcgov/entity/issues/22396

*Description of changes:*
- Create new `useApplications` composable
- Use new `post` and `get` endpoints for `/applications`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-81-jbt3qtol.web.app

File: strr-web/composables/useChipFlavour.ts
Comment (rstens) [inline strr-web/composables/useChipFlavour.ts:1]: Can we align the "SUBMITTED" and "APPLIED" , it now is a mix.

Comment (rstens) [reply strr-web/composables/useChipFlavour.ts:1]: export enum AlertsFlavourE {
  ALERT = 'alert',
  INFO = 'info',
  WARNING = 'warning',
  SUCCESS = 'success',
  MESSAGE = 'message',
  APPLIED = 'pending'
}
I am confused

Comment (dimak1) [reply strr-web/composables/useChipFlavour.ts:1]: I will need to refactor them, as these statuses are mixed between Applications and Registrations. So, it's a work in progress. Thanks for noticing.

Comment (dimak1) [reply strr-web/composables/useChipFlavour.ts:1]: The flavours are the styling for the chips/badges from what I understand.

**Pull request merged**